README.md

```diff
@@ -45,13 +44,13 @@ For each library you wish to use, add an `implementation` configuration to the `
 // build.gradle
 dependencies {
     ...
-    implementation "com.azure.android:azure-communication-chat:1.0.0-beta.6"
+    implementation "com.azure.android:azure-communication-chat:1.0.1"
```

Comment:
Is there a corresponding change for Common? These two goes hand-in-hand 'cause there is no use for common other than Chat